### DISCUSSION TITLE
Possibility to use Node (NPM) modules?

### DISCUSSION THREAD

**Discussion by dennis2848** [SuperOffice/typescript-for-superoffice#14] [General]
Hey, i have been looking into the examples, and see that all the examples only use the @superoffice/webapi package as well as context - so a question came into my mind

Will there at some point be support for importing node modules, or is that already possible somehow?

I can see many instances where a node module would make certain tasks much easier to accomplish, but it would require some kind of bundler i guess? Otherwise there would probably be quite a lot of "reinventing the wheel" kind of coding.

Any thoughts about this? 

Good work so far.

//Dennis - DoWEB

**Comment by ejfasting** [comment]
Hello @dennis2848 

Sorry for taking to long to get back to you.. 

Importing your own npm modules would be a great feature! We WISH to support this, but we are currently unsure what security implications this would introduce, as we cant validate that the NPM modules you import do not contain malicious code. 
So, for now, its not supported, but I will bring this feedback into the discussion when we get back to it :)

Great that you brought it up!

//Eivind
